Headers/DebugServer2/Architecture/ARM/HardwareBreakpointManager.h

```diff
@@ -23,25 +23,25 @@ class HardwareBreakpointManager : public BreakpointManager {
   ~HardwareBreakpointManager() override;
 
 public:
-  virtual ErrorCode add(Address const &address, Type type, size_t size,
-                        Mode mode) override {
+  ErrorCode add(Address const &address, Type type, size_t size,
```

Comment:
`override` implies `virtual`, so your commit is good, but I'm not sure it does what you think it does.